docs/features/tcp-health-probe.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "TCP Health probe"
+layout: default
+---
+
+# TCP Health probe
+
+A `BackgroundService` can be added to the <span>ASP.NET</span> hosted services to expose a TCP health endpoint that allows a runtime to periodically check the liveness/readiness of the host.
+
+## Installation
+
+This features requires to install our NuGet package:
+
+```shell
+PM > Install-Package Arcus.Messaging.Health
+```
+
+## Usage
+
+To include the TCP endpoint, add the following line of code in the `Startup.ConfigureServices` method:
+
+```csharp
+public void ConfigureServices(IServiceCollection services)
+{
+    services.AddTcpHealthProbes(healthBuilder => 
+    {
+        healthBuilder.AddCheck("Example", () => HealthCheckResult.Healthy("Example is OK!"), tags: new[] { "example" })
```

Comment:
Maybe mention that this is optional and you can just do `services.AddTcpHealthProbes()`?

Reply:
Yep, added this in the inline code now, with comments explaining.